_install-and-configure/install-dashboards/plugins.md

```diff
@@ -28,132 +28,82 @@ Major, minor, and patch plugin versions must match OpenSearch major, minor, and
 ```
+{% include copy.html %}
 
-This plugin lets you combine OpenSearch Dashboards visualizations and narrative text in a single interface.
-
-#### Reports OpenSearch Dashboards
-
-```bash
-# x86 Linux
-sudo bin/opensearch-dashboards-plugin install https://d3g5vo6xdbdb9a.cloudfront.net/downloads/opensearch-dashboards-plugins/opensearch-reports/linux/x64/opensearchReportsOpenSearch Dashboards-{{site.opensearch_major_minor_version}}.2.0-linux-x64.zip
-# ARM64 Linux
-sudo bin/opensearch-dashboards-plugin install https://d3g5vo6xdbdb9a.cloudfront.net/downloads/opensearch-dashboards-plugins/opensearch-reports/linux/arm64/opensearchReportsOpenSearch Dashboards-{{site.opensearch_major_minor_version}}.2.0-linux-arm64.zip
-# x86 Windows
-sudo bin/opensearch-dashboards-plugin install https://d3g5vo6xdbdb9a.cloudfront.net/downloads/opensearch-dashboards-plugins/opensearch-reports/windows/x64/opensearchReportsOpenSearch Dashboards-{{site.opensearch_major_minor_version}}.2.0-windows-x64.zip
-```
-
-This plugin lets you export and share reports from OpenSearch Dashboards dashboards, visualizations, and saved searches.
+## Viewing a list of installed plugins
 
-#### Gantt Chart OpenSearch Dashboards
+To view the list of installed plugins from the command line, use the following command:
 
 ```bash
-sudo bin/opensearch-dashboards-plugin install https://d3g5vo6xdbdb9a.cloudfront.net/downloads/opensearch-dashboards-plugins/opensearch-gantt-chart/opensearchGanttChartOpenSearch Dashboards-{{site.opensearch_major_minor_version}}.0.0.zip
+sudo bin/opensearch-dashboards-plugin list
 ```
+{% include copy.html %}
 
-This plugin adds a new Gantt chart visualization.
-
-{% endcomment %}
-
-## Viewing a list of installed plugins
-
-To view the list of installed plugins from the command line, use the following command:
+The command returns the list of installed plugins and their versions:
 
 ```bash
-sudo bin/opensearch-dashboards-plugin list
+alertingDashboards@3.1.0.0
+anomalyDetectionDashboards@3.1.0.0
+assistantDashboards@3.1.0.0
+customImportMapDashboards@3.1.0.0
+flowFrameworkDashboards@3.1.0.0
+indexManagementDashboards@3.1.0.0
+mlCommonsDashboards@3.1.0.0
+notificationsDashboards@3.1.0.0
+observabilityDashboards@3.1.0.0
+queryInsightsDashboards@3.1.0.0
+queryWorkbenchDashboards@3.1.0.0
+reportsDashboards@3.1.0.0
+searchRelevanceDashboards@3.1.0.0
+securityAnalyticsDashboards@3.1.0.0
 ```
 
 ## Remove plugins
 
-To remove a plugin:
+To remove a plugin, use the following command:
 
 ```bash
-sudo bin/opensearch-dashboards-plugin remove <plugin-name>
+sudo bin/opensearch-dashboards-plugin remove alertingPlugin
 ```
+{% include copy.html %}
 
 Then remove all associated entries from `opensearch_dashboards.yml`.
 
-For certain plugins, you must also remove the "optimize" bundle. This is a sample command for the Anomaly Detection plugin:
+For certain plugins, you must also remove the optimized bundle. This is a sample command for the Anomaly Detection plugin:
```

Comment:
@peterzhuamazon Could you clarify what plugins require this step so users know?

Reply:
https://github.com/opensearch-project/opensearch-build/blob/main/manifests/3.1.0/opensearch-dashboards-3.1.0.yml

Reply:
On a glance this is correct compares to the manifest, since OSD core + FTrepo is not plugins and the list is complete.